### ISSUE TITLE
Security Certificate Request

### ISSUE THREAD

**Issue by rmtolmach** [department-of-veterans-affairs/va.gov-team#51360]
# Description
_To be done after https://github.com/department-of-veterans-affairs/va.gov-team/issues/50151_

As part of the ESECC request, an SSL certificate needs to be requested. From the ESECC request: 
> Provide the SSL certificate in pFX or P12 format and send a separate encrypted email providing the password to split the file. TLS 1.1 and above is required.

Once an ESECC is approved, the certificates will need to be installed. Information about this process can be found in this document: https://vfs.atlassian.net/wiki/spaces/CLOUD/pages/2231926890/ESECC+Request+-+Cloud+Isolation#Security-Certificate-Request **and in the instructions linked there titled "Obtain or Renew Publicly Signed SSL Certificates"** 

And here are the official instructions written by Boris (there is some overlap between these and the ones in Confluence pasted above): https://github.com/department-of-veterans-affairs/devops/blob/master/docs/ESECC-Public-URL-process.md#ssl-cert-with-nsoc-gateway-ops 

# Tasks
- [x] Set up SSLMate
- [ ] Purchase certificates for all of the domains (`dev-preview.va.gov` and `preview.va.gov`)
- [ ] Send an email to NOC Network Edge Operations NOCNetworkEdgeOperations@va.gov, with a cc to VA NSOC Operations (GateWay) VANSOCOperationsGateWay@va.gov, Bill Chapman William.Chapman7@va.gov, Ray Wang gang.wang@va.gov and Mike Chelen Michael.Chelen@va.gov to indicate that someone should be monitoring the `admin@va.gov` account to validate the domain ownership for the SSL certificates.
- [ ] Once domain is verified, continue following these steps: https://vfs.atlassian.net/wiki/spaces/OT/pages/1644625921/Obtain+or+Renew+Publicly+Signed+SSL+Certificates#Process-to-acquire-a-new-certificate

# Acceptance Criteria
- [x] Certificate(s) have been purchased (with credit card on sslmate file)
- [x] Email sent to notify the correct parties to validate ownership.
- [x] Ownership has been verified by admin
- [x] Certificate in P12 format has been sent to VANSOCOperationsGateWay@va.gov (instructions are in [Process to acquire a new certificate](https://vfs.atlassian.net/wiki/spaces/OT/pages/1644625921/Obtain+or+Renew+Publicly+Signed+SSL+Certificates#Process-to-acquire-a-new-certificate))

**Comment by rmtolmach** [comment]
I didn't do any of the steps to start this process, but I did learn from Andrea, who's gone through this process before, that there is a credit card on file with the `greggersh` sslmate account, so that will be used automatically when purchasing the certificate. The password for the `greggersh` account is stored in Parameter Store (instructions in doc), which Cyril has access to.

**Comment by pjhill** [comment]
Purchase information submitted to Ray and Andrea -- awaiting review

**Comment by pjhill** [comment]
Ray has approved. Andrea is attempting to allocate funding from a particular bucket and will notify when we are able to proceed.

**Comment by pjhill** [comment]
We are still awaiting a response from Enterprise Gateway group. We need someone who monitors admin@va.gov to confirm the purchase of the subdomain certificates.

**Comment by pjhill** [comment]
Sent a reminder email yesterday -- no response yet from a party with access to admin@va.gov.

**Comment by pjhill** [comment]
No response today -- the new plan is to ask Ray to ping people on Monday.

**Comment by pjhill** [comment]
Certificates have been downloaded. The process to submit the certificates has been clarified and a ServiceNow ticket has been created.
